### DISCUSSION TITLE
feat: 云黑记录功能

### DISCUSSION THREAD

**Discussion by HsiangNianian** [HydroRoll-Team/support#9] [Polls]
> 是否支持云黑记录是一个很难做决定的事情，因此让大家投票看看究竟意愿如何。

**Comment by M1oFeather** [comment]
云黑应该做成可选择开启的功能，让骰主视自身骰娘情况开启或关闭

**Comment by mystringEmpty** [comment]
所有Bot都可以公开一个端口分享黑名单，也可以主动选择自动同步哪些来源的黑名单

**Reply by HsiangNianian** [reply]
great

**Reply by fu050409** [reply]
开放端口会导致不必要的渗透风险，这可能会被黑客利用。同时，在国内的大内网环境下，相当一部分骰主的服务器不包含公网地址。这意味着这个功能是十分局限的。

**Comment by HTL29** [comment]
咱认为还是需要的，不过要取决于使用者
